vault/identity_store_util.go

```diff
@@ -341,6 +341,12 @@ func (i *IdentityStore) loadEntities(ctx context.Context) error {
 					}
 				}
 
+				duplicateAccessors := getDuplicateAccessorsOnAliases(ctx, entity.Aliases)
+
+				// Log a warning if an entity has multiple aliases with the same accessor
+				if len(duplicateAccessors) > 0 {
+					i.logger.Warn("Duplicate aliases present for the same entity and accessor", "entity_name", entity.Name, "namespace_id", entity.NamespaceID, "duplicate_accessors", duplicateAccessors)
```

Comment:
are we concerned that this is leaking any privileged information?

Reply:
Hmm I don't think so? It logs some entity information and mount UUIDs, which afaik aren't sensitive, is there a specific field of the ones logged that concerns you more?